_data/finance.yml

```diff
@@ -239,7 +239,9 @@ websites:
       url: https://www.northwesternmutual.com/
       facebook: northwesternmutual
```

Comment:
Please remove their Facebook handle as it's not needed if they support 2FA.
```suggestion
```

Reply:
Latest commit removes the Facebook handle